src/Booma.Proxy.Packets.PatchServer/Payloads/Server/PatchingFileChunkPayload.cs

```diff
@@ -7,10 +7,6 @@
 
 namespace Booma.Proxy
 {
-	//0x0C 0x00 0x11 0x00
-	//PatchingByteLength{4} PatchFileCount{4}
-	//Tethella implementation: https://github.com/justnoxx/psobb-tethealla/blob/master/patch_server/patch_server.c#L578
-	//Sylverant implementation: https://github.com/Sylverant/patch_server/blob/master/src/patch_packets.c#L237 and structure https://github.com/Sylverant/patch_server/blob/master/src/patch_packets.h#L106
```

Comment:
Why are you removing the documentation I added for this packet?

Reply:
Oh nvm, this was incorrect documentation. Must be from copypaste

Reply:
Yeah, sorry, it was copy paste stuff I forgot to remove right away.